itest/lnd_misc_test.go

```diff
@@ -1344,3 +1347,51 @@ func testSendSelectedCoinsChannelReserve(ht *lntest.HarnessTest) {
 	// Alice should have one reserved UTXO now.
 	ht.AssertNumUTXOs(alice, 1)
 }
+
+// testGRPCNotFound verifies that the expected grpc NotFound status code is
+// returned when querying various rpc endpoints.
+func testGRPCNotFound(ht *lntest.HarnessTest) {
+	var (
+		notFoundErr = codes.NotFound.String()
+		unknownPub  = "0286098b97bc843372b4426d4b276cea9aa2f48f0428d6" +
+			"f5b66ae101befc14f8b4"
+		rHash = make([]byte, 32)
+	)
+	unknownPubBytes, err := route.NewVertexFromStr(unknownPub)
+	require.NoError(ht, err)
+
+	_, err = rand.Read(rHash)
+	require.NoError(ht, err)
```

Comment:
nit: just fyi there are `ht.Random32Bytes()` and `ht.RandomPreimage()`

Reply:
cool - gonna do this in a commit in the migration plugin /itest follow up PR I open today 👍